pkgs/applications/editors/vim/plugins/update.py

```diff
@@ -123,6 +131,18 @@ def main():
     args = parser.parse_args()
     pluginupdate.update_plugins(editor, args)
 
+    if editor.nvim_treesitter_updated:
+        print("updating nvim-treesitter grammars")
+        nvim_treesitter_dir = ROOT.joinpath("nvim-treesitter")
+        subprocess.check_call([nvim_treesitter_dir.joinpath("update.py")])
+
+        if editor.nixpkgs_repo:
+            msg = "vimPlugins.nvim-treesitter: update grammars"
+            print(f"committing to nixpkgs: {msg}")
+            index = editor.nixpkgs_repo.index
+            index.add([str(nvim_treesitter_dir.joinpath("generated.nix"))])
+            index.commit(msg)
```

Comment:
I think this needs the same none commit flag as the other script because PythonGit has a bug that sometimes commiting just fails.

Reply:
`nixpkgs_repo` is `None` if `autocommit` is false